tstore/tslong/dask.py

```diff
@@ -1,16 +1,125 @@
 """Module defining the TSLongDask wrapper."""
 
+from typing import TYPE_CHECKING
+
+import pandas as pd
+
+from tstore.tsdf.ts_class import TS
+from tstore.tsdf.tsarray import TSArray
+from tstore.tslong.pyarrow import TSLongPyArrow
 from tstore.tslong.tslong import TSLong
 
+if TYPE_CHECKING:
+    # To avoid circular imports
+    from tstore.tsdf.dask import TSDFDask
+    from tstore.tswide.dask import TSWideDask
+
 
 class TSLongDask(TSLong):
     """Wrapper for a long-form Dask timeseries dataframe."""
 
-    def to_tstore(self):
+    def to_tstore(
+        self,
+        base_dir,
+        partitioning=None,
+        tstore_structure="id-var",
+        overwrite=True,
+    ):
         """Write the wrapped dataframe as a TStore structure."""
-        raise NotImplementedError
+        pandas_tslong = self.change_backend(new_backend="pandas")
```

Comment:
Ouch. If the dask dataframe is very large, it might be worth to group by `store_id` first and then convert group by group to pandas to avoid memory explosion (and append to tstore).
BTW: We don't allow yet to append to tstore right? Also only in terms of store_id?